demo/src/screens/componentScreens/HapticScreen.tsx

```diff
@@ -2,11 +2,11 @@ import _ from 'lodash';
 import React, {Component} from 'react';
 import {ScrollView} from 'react-native';
 import {View, Text, Button} from 'react-native-ui-lib';
-import {HapticMethods, triggerHaptic} from '../../../../src/services';
+import {HapticService} from '../../../../src/services';
```

Comment:
I missed something 😬 
You should also export the HapticService in the main src/index and use it from there